apps/frontend/src/components/ShowMembers/ShowMembers.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import ReactDOM from "react-dom/client";
+import "./index.css";
+
+const usernames = ["radu", "antonina", "ecem", "maria", "mike", "john"];
+
+function App() {
+  return (
+    <div>
+      <UserCircleStack />
+    </div>
+  );
+}
+
+function UserCircleStack() {
```

Comment:
This component should be able to take usernames as **props**, just like you did with the **UserCircle** component!

Reply:
I changed it as you mentioned.